docs/getting-started/quickstart.md

```diff
@@ -93,157 +117,89 @@ crate-type = ["cdylib"]
 
 ```bash
 cargo build --release --target wasm32-unknown-unknown
 ```
 
-Start the project by using the [`run`](https://substreams.streamingfast.io/reference-and-specs/command-line-interface#run) command:
+### 5. Execute
+
+To execute, or run, the example use the `substreams` [`run`](https://substreams.streamingfast.io/reference-and-specs/command-line-interface#run) command:
 
 {% code overflow="wrap" %}
+
 ```bash
-substreams run -e mainnet.eth.streamingfast.io:443 substreams-ethereum-tutorial.yaml map_basic_eth --start-block 1000000 --stop-block +1
+substreams run -e mainnet.eth.streamingfast.io:443 substreams-ethereum-tutorial.yaml map_basic_eth --start-block 10000001 --stop-block +1
 ```
-{% endcode %}
-
-### Solana example
-
-The procedure to use the Solana example is very similar to the Ethereum example.&#x20;
-
-Clone or download the Solana example codebase. The example is located in the official [Substreams Solana Tutorial GitHub repository](https://github.com/streamingfast/substreams-solana-tutorial).
-
-After cloning the Solana example, take a moment to look through the repository. Differences from the Ethereum example stand out immediately.
 
-A very important distinction between the two examples is the module handler code. Different blockchains have their own architectures, implementations, and data structures. Blocks for Ethereum, Solana, or even Bitcoin are constructed and designed differently. Some of the differences are small and subtle, others are not.
-
-Notice the Solana module handler uses the `previous_blockhash`, `blockhash,` and `slot` fields of the _`block`_ passed into the handler by Substreams. The Ethereum example's module handler uses the `ver` and `number` fields of the _`block`_ it receives. The disparities in the field names are due to differences in the block model for the separate blockchains.
-
-{% code title="src/lib.rs" overflow="wrap" %}
-```rust
-#[substreams::handlers::map]
-fn map_basic_sol(block: solpb::sol::v1::Block) -> Result<basicexample::BasicExampleProtoData, substreams::errors::Error> {
-    log::info!("block.previous_blockhash: {:#?}", block.previous_blockhash);
-    log::info!("block.blockhash: {:#?}", block.blockhash);
-    log::info!("block.slot: {:#?}", block.slot);
-    Ok(basicexample::BasicExampleProtoData {blockhash: block.blockhash})
-}
-```
 {% endcode %}
 
-Follow the same steps [used for the Ethereum example](quickstart.md#ethereum-example) to [`run`](https://substreams.streamingfast.io/reference-and-specs/command-line-interface#run) the Solana example.
+To ensure that the [`run`](https://substreams.streamingfast.io/reference-and-specs/command-line-interface#run) command is executed correctly, you need to pass the proper endpoint, manifest name, and module handler name. The `--start-block` and `--stop-block` flags are optional, but they can help limit the results that are returned to the client.
```

Comment:
Feels like it's not important for the quick start, should probably be moved in `Next Steps` for those you would maybe want to test a Substreams for another chain.

Reply:
Updated.